katalog/istio/init/README.md

```diff
@@ -0,0 +1,40 @@
+# Init Istio package
+
+Package containing Istio CRDs. Needed to install istio on top of it.
```

Comment:
I think "Needed to install istio on top of it." doesn't add much. maybe refrase to:
"This is required to run istio in your cluster".

Reply:
Solved!